sim/src/lib.rs

```diff
@@ -9,6 +9,7 @@ mod scheduler;
 mod sim;
 mod transit;
 mod trips;
+mod pmodel;
```

Comment:
Could we organize `pandemic.rs` and `pmodel.rs` into a new subdirectory called `pandemic` and just expose `PandemicModel` from there? More room to grow

Reply:
yup, i'll do that